docs/sources/enterprise/http_api.md

```diff
@@ -0,0 +1,72 @@
++++
+title = "Enterprise HTTP APIs"
+description = "Grafana Enterprise APIs"
+keywords = ["grafana", "enterprise", "api"]
+aliases = ["/docs/grafana/latest/enterprise/http_api/"]
+type = "docs"
+[menu.docs]
+name = "HTTP APIs"
+parent = "enterprise"
+weight = 1000
++++
+
+# Enterprise HTTP APIs
+
+## Authentication
+
+To access the API you need to authenticate yourself. For more information about API authentication, refer to [Authentication API]({{< relref "../http_api/auth.md" >}}).
+
+## Reporting API
+
+### Send report
+
+> Only available in Grafana Enterprise v7.0+.
+
+`POST /api/reports/email`
+
+Generate and send a report. This API waits for the report to be generated before returning. You are recommended to set the client's timeout to 60 seconds or more.
+
+This API endpoint is experimental and may be deprecated in a future release. On deprecation, a migration strategy will be provided and the endpoint will remain functional until the next major release of Grafana.
```

Comment:
You need to lead with this. Is it just reports/email that is experimental, or is it the whole reporting API? Put it as high up in the hierarchy as makes sense.

Reply:
Just the `POST /api/reports/email` endpoint.